src/etc/module.xml

```diff
@@ -17,6 +17,7 @@
             <module name="Magento_Customer" />
             <module name="Magento_Wishlist" />
             <module name="Magento_UrlRewrite" />
+            <module name="ScandiPWA_Customization"/>
```

Comment:
Magento outputs error message when running command "setup:upgrade" : "_Circular sequence reference from 'ScandiPWA_Route717' to 'ScandiPWA_Customization'._" as ScandiPWA_Customization is referencing ScandiPWA_Route717 by proxy... solution worked without this line.